chef/cookbooks/apache2/recipes/mod_python.rb

```diff
@@ -17,14 +17,11 @@
 # limitations under the License.
 #
 
-case node[:platform]
-  when "debian", "ubuntu"
-    package "libapache2-mod-python" do
-      action :install
-    end
-  when "centos", "redhat", "fedora"
+case node[:platform_family]
+  when "debian"
+    package "libapache2-mod-python"
+  when "rhel", "fedora"
```

Comment:
Style/CaseIndentation: Indent `when` as deep as `case`. (https://github.com/bbatsov/ruby-style-guide#indent-when-to-case)